polkadot/runtime/parachains/src/configuration/migration/v11.rs

```diff
@@ -264,7 +264,7 @@ mod tests {
 			let mut configs_to_check = v11::PendingConfigs::<Test>::get().unwrap();
 			configs_to_check.push((0, v11.clone()));
 
-			for (_, v10) in configs_to_check {
+			for (_, v11) in configs_to_check {
```

Comment:
good catch. Maybe we should move the v11 above into its own scope {} with that assert.